Performance tests failing llm_performance_grpo_qwen3_30ba3b_4n8g_40K

Perf test failing off main branch llm_performance_grpo_qwen3_30ba3b_4n8g_40K with this error:
```

 (VllmGenerationWorker pid=1969904, ip=10.65.0.85) INFO 06-13 21:55:29 [abstract.py:349] It took 0.627090 seconds to wake up tags ['weights'].
Traceback (most recent call last):
  File "/lustre/fs1/portfolios/coreai/projects/coreai_dlalgo_genai/nemo_rl_ci/main/54649972/RL/code_snapshots/grpo-qwen3-30ba3b-4n8g-40K/examples/run_grpo.py", line 225, in <module>
    main()
    ~~~~^^
  File "/lustre/fs1/portfolios/coreai/projects/coreai_dlalgo_genai/nemo_rl_ci/main/54649972/RL/code_snapshots/grpo-qwen3-30ba3b-4n8g-40K/examples/run_grpo.py", line 208, in main
    trainer(
    ~~~~~~~^
        policy,
        ^^^^^^^
    ...<10 lines>...
        master_config,
        ^^^^^^^^^^^^^^
    )
    ^
  File "/lustre/fs1/portfolios/coreai/projects/coreai_dlalgo_genai/nemo_rl_ci/main/54649972/RL/code_snapshots/grpo-qwen3-30ba3b-4n8g-40K/nemo_rl/algorithms/grpo.py", line 1911, in grpo_train
    refit_policy_generation(
    ~~~~~~~~~~~~~~~~~~~~~~~^
        policy,
        ^^^^^^^
    ...<3 lines>...
        kv_scales=kv_scales_cache if sync_kv_scales else None,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/lustre/fs1/portfolios/coreai/projects/coreai_dlalgo_genai/nemo_rl_ci/main/54649972/RL/code_snapshots/grpo-qwen3-30ba3b-4n8g-40K/nemo_rl/algorithms/grpo.py", line 1540, in refit_policy_generation
    ray.get(futures_train)
    ~~~~~~~^^^^^^^^^^^^^^^
  File "/opt/nemo_rl_venv/lib/python3.13/site-packages/ray/_private/auto_init_hook.py", line 22, in auto_init_wrapper
    return fn(*args, **kwargs)
  File "/opt/nemo_rl_venv/lib/python3.13/site-packages/ray/_private/client_mode_hook.py", line 107, in wrapper
    return func(*args, **kwargs)
  File "/opt/nemo_rl_venv/lib/python3.13/site-packages/ray/_private/worker.py", line 2980, in get
    values, debugger_breakpoint = worker.get_objects(
                                  ~~~~~~~~~~~~~~~~~~^
        object_refs, timeout, use_object_store=_use_object_store
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/nemo_rl_venv/lib/python3.13/site-packages/ray/_private/worker.py", line 1023, in get_objects
    raise value.as_instanceof_cause()
ray.exceptions.RayTaskError(AssertionError): ray::MegatronPolicyWorker.stream_weights_via_ipc_zmq() (pid=410174, ip=10.65.0.149, actor_id=04643eccd2009981a105bc1e01000000, repr=MegatronPolicyWorker[rank=4])
  File "/opt/ray_venvs/nemo_rl.models.policy.workers.megatron_policy_worker.MegatronPolicyWorker/lib/python3.13/site-packages/torch/utils/_contextlib.py", line 124, in decorate_context
    return func(*args, **kwargs)
  File "/lustre/fs1/portfolios/coreai/projects/coreai_dlalgo_genai/nemo_rl_ci/main/54649972/RL/code_snapshots/grpo-qwen3-30ba3b-4n8g-40K/nemo_rl/utils/nsys.py", line 88, in wrapper
    ret = func(*args, **kwargs)
  File "/lustre/fs1/portfolios/coreai/projects/coreai_dlalgo_genai/nemo_rl_ci/main/54649972/RL/code_snapshots/grpo-qwen3-30ba3b-4n8g-40K/nemo_rl/models/policy/workers/megatron_policy_worker.py", line 1340, in stream_weights_via_ipc_zmq
    stream_weights_via_ipc_zmq_impl(
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        params_generator=self._iter_params_with_optional_kv_scales(
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ...<5 lines>...
        worker_name=str(self),
        ^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/lustre/fs1/portfolios/coreai/projects/coreai_dlalgo_genai/nemo_rl_ci/main/54649972/RL/code_snapshots/grpo-qwen3-30ba3b-4n8g-40K/nemo_rl/models/policy/utils.py", line 344, in stream_weights_via_ipc_zmq_impl
    assert aligned_size <= buffer_size_bytes, (
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: Parameter model.embed_tokens.weight too large for buffer: 622329856 > 593500569

```

--- a/examples/configs/grpo_math_qwen30ba3b_megatron.yaml
+++ b/examples/configs/grpo_math_qwen30ba3b_megatron.yaml
@@ -73,6 +73,10 @@ policy:
       gpu_memory_utilization: 0.7
       enforce_eager: false
       max_model_len: ${policy.max_total_sequence_length}
+    vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
+      moe_backend: triton
 
 cluster:
   gpus_per_node: 8

--- a/examples/configs/recipes/llm/grpo-dapomath17k-dsv3-megatron.yaml
+++ b/examples/configs/recipes/llm/grpo-dapomath17k-dsv3-megatron.yaml
@@ -40,6 +40,10 @@ policy:
     vllm_cfg:
       async_engine: true
       tensor_parallel_size: 32
+    vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
+      moe_backend: triton
 data:
   train:
     dataset_name: DAPOMath17K

--- a/examples/configs/recipes/llm/grpo-gptoss-20b-8n8g-megatron.yaml
+++ b/examples/configs/recipes/llm/grpo-gptoss-20b-8n8g-megatron.yaml
@@ -22,6 +22,8 @@ policy:
     vllm_cfg:
       tensor_parallel_size: 2
     vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
       moe_backend: triton
 cluster:
   num_nodes: 8

--- a/examples/configs/recipes/llm/grpo-math-qwen3-30ba3b-megatron-tp4-32k.yaml
+++ b/examples/configs/recipes/llm/grpo-math-qwen3-30ba3b-megatron-tp4-32k.yaml
@@ -38,6 +38,10 @@ policy:
     vllm_cfg:
       tensor_parallel_size: 4
       enforce_eager: true
+    vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
+      moe_backend: triton
 logger:
   log_dir: logs/grpo-math-qwen3-30ba3b-megatron-tp4-32k
   wandb_enabled: true

--- a/examples/configs/recipes/llm/grpo-qwen3-30ba3b-8n8g-megatron-cp2-r3.yaml
+++ b/examples/configs/recipes/llm/grpo-qwen3-30ba3b-8n8g-megatron-cp2-r3.yaml
@@ -27,7 +27,6 @@ policy:
       enable_prefix_caching: false
     vllm_kwargs:
       enable_chunked_prefill: false
-      moe_backend: triton
 data:
   max_input_seq_length: 2048
 data_plane:

--- a/examples/configs/recipes/llm/grpo-qwen3-30ba3b-8n8g-megatron.yaml
+++ b/examples/configs/recipes/llm/grpo-qwen3-30ba3b-8n8g-megatron.yaml
@@ -36,6 +36,10 @@ policy:
     vllm_cfg:
       tensor_parallel_size: 4
       gpu_memory_utilization: 0.7
+    vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
+      moe_backend: triton
 logger:
   log_dir: logs/grpo-qwen3-30ba3b-8n8g-megatron
   wandb_enabled: true

--- a/examples/configs/recipes/llm/performance/grpo-qwen3-30ba3b-4n4g.yaml
+++ b/examples/configs/recipes/llm/performance/grpo-qwen3-30ba3b-4n4g.yaml
@@ -36,6 +36,10 @@ policy:
   generation:
     vllm_cfg:
       tensor_parallel_size: 1
+    vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
+      moe_backend: triton
 logger:
   log_dir: logs/grpo-qwen3-30ba3b-4n4g
   wandb_enabled: true

--- a/examples/configs/recipes/llm/performance/grpo-qwen3-30ba3b-4n8g-40K.yaml
+++ b/examples/configs/recipes/llm/performance/grpo-qwen3-30ba3b-4n8g-40K.yaml
@@ -39,6 +39,10 @@ policy:
   generation:
     vllm_cfg:
       tensor_parallel_size: 2
+    vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
+      moe_backend: triton
 logger:
   log_dir: logs/grpo-qwen3-30ba3b-4n8g
   wandb_enabled: true

--- a/examples/configs/recipes/llm/performance/grpo-qwen3-30ba3b-4n8g.yaml
+++ b/examples/configs/recipes/llm/performance/grpo-qwen3-30ba3b-4n8g.yaml
@@ -36,6 +36,10 @@ policy:
   generation:
     vllm_cfg:
       tensor_parallel_size: 2
+    vllm_kwargs:
+      # vLLM 0.20's default FlashInfer TRTLLM MoE backend isn't refit-compatible,
+      # change to triton to keep same behavior as vLLM 0.17.
+      moe_backend: triton
 logger:
   log_dir: logs/grpo-qwen3-30ba3b-4n8g
   wandb_enabled: true

